sale_order_color_code_tintometric_system_stock/reports/stock_picking_template.xml

```diff
@@ -0,0 +1,23 @@
+<odoo>
+    <template id="report_delivery_document" inherit_id="stock.report_delivery_document">
+        <xpath expr="//th[@name='th_sm_product']" position="after">
+            <t t-set="display_color" t-value="any(line.color_code_tintometric for line in o.move_ids_without_package)"/>
```

Comment:
color_code_tintometric, when suggested related implementation is applied is directly a sale.order.line field. Since an Inventory user shouldn't sometimes have access to Sales, sudo usage is required:
```suggestion
            <t t-set="display_color" t-value="any(line.sudo().color_code_tintometric for line in o.move_ids_without_package)"/>
```